examples/kotlin/BUILD.bazel

```diff
@@ -21,28 +23,66 @@ kt_js_library(
     deps = [":kotlinx-html-js"],
 )
 
+# Copy boostrap to the bin folder as entry_point.js
+# so that relative import to `./hello.js` is valid
+genrule(
+    name = "bootstrap",
+    srcs = ["bootstrap.js"],
+    outs = ["entry_point.js"],
+    cmd = "cp $< $@",
+)
+
 rollup_bundle(
     name = "bundle",
     srcs = ["hello.js"],
-    entry_point = "bootstrap.js",
+    config_file = "rollup.config.js",
+    entry_point = "entry_point.js",
+    # TODO: make this example work with format = "esm"
+    format = "cjs",
+    output_dir = True,
     deps = [
         "@npm//kotlin",
         "@npm//kotlinx-html-js",
     ],
 )
 
+babel(
```

Comment:
no need to have diff. loading in this example

Reply:
agreed